test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist.py

```diff
@@ -118,3 +125,45 @@ def _test_mnist_distributed(
         output = predictor.predict(data)
 
         assert output.shape == (batch_size, 10)
+
+        #  Check for Cloudwatch logs if true
+        if verify_logs:
+            _check_for_cloudwatch_logs(endpoint_name)
+
+def _check_for_cloudwatch_logs(endpoint_name):
+    print('##############################################################################')
+    print('##############################################################################')
+    print('##############################################################################')
+    print('INFO: Checking logs for the endpoint: /aws/sagemaker/'+endpoint_name)
+    print('##############################################################################')
+    print('##############################################################################')
+    print('##############################################################################')
+    print('##############################################################################')
+    client=boto3.client('logs')
+    query = "fields @timestamp | sort @timestamp desc | limit 2";
+    start_query_response = client.start_query(
+        logGroupName='/aws/sagemaker/'+endpoint_name,
+        startTime=int((datetime.today() - timedelta(minutes=4)).timestamp()),
+        endTime=int(datetime.now().timestamp()),
+        queryString=query,
+    )
+    query_id = start_query_response['queryId']
+    response = None
+    while response == None or response['status'] == 'Running':
+        print('Waiting for query to complete ...')
+        time.sleep(1)
+        response = client.get_query_results(
+            queryId=query_id
+        )        
+    recordsAvailable=bool(response['results'])
+    if not recordsAvailable:
+        print("Exception... No results")
```

Comment:
you can just raise exception here - will be easier to see logs

Reply:
done